internal/table/table.go

```diff
@@ -175,7 +175,12 @@ func renderRow(providedColumns []string, colWidths []int) string {
 			}
 
 			suffix := strings.Repeat(" ", maxLen-end)
-			result += pad(string(colValue[0:end]) + suffix)
+			if len(colValue) > end && colValue[end-1] == linebreakRune {
+				result += pad(string(colValue[0:end-1]) + suffix)
+			} else {
+				result += pad(string(colValue[0:end]) + suffix)
+			}
+
```

Comment:
I'm unsure if this is the correct way to handle this, though it did fix my problem. Please let me know if there is a better approach

Reply:
Maybe we could do something like this:

```
			skipLineBreak := 0
			if breakpos := runeSliceIndexOf(colValue, linebreakRune); breakpos != -1 && breakpos < end {
				end = breakpos + 1
				skipLineBreak = 1
			}
 			suffix := strings.Repeat(" ", maxLen-end)
 			result += pad(string(colValue[0:end-skipLineBreak]) + suffix) 

```

Reply:
That would get rid of the second if-clause